dbfit-java/build.gradle

```diff
@@ -222,3 +251,116 @@ project('netezza') {
     }
 }
 
+// Aggregated dependencies to include in the POM file when publishing.
+dependencies {
```

Comment:
I tried, very hard, to extend the dependency configurations of the parent `dbfit-java` project from it's child projects (`core`, etc) but I was always presented with null pointer exceptions so I had to fall back to specifying the same dependencies for the parent `dbfit-java` project and the child projects using the artefact lists. Intuitively this sounds like the wrong way of trying to inherit anyway.

I'm not sure what you mean by `tag`. What is that?